content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+└── 🔐 Signatures for an image tag: ghcr.io/falcosecurity/plugins/plugin/cloudtrail:sha256-ec47c7448d455b6bb84f39d5c400c2fa207ab5fbe8c21327c6c1d06a3f1eecb4.sig
+   └── 🍒 sha256:f8aae8441b2d41419ae99d4979d6eab1543d81a269e4721fdbff0d837571aa4d
+```
+
+And you can inspect with the [`oras`](https://oras.land/docs/category/oras-commands/) CLI its manifest, by taking the tag that contains the digest content identifier of the artifact signed (i.e. `ghcr.io/falcosecurity/plugins/plugin/cloudtrail:sha256-ec47c7448d455b6bb84f39d5c400c2fa207ab5fbe8c21327c6c1d06a3f1eecb4.sig`):
+
+```json=
+$ oras manifest fetch ghcr.io/falcosecurity/plugins/plugin/cloudtrail:sha256-ec47c7448d455b6bb84f39d5c400c2fa207ab5fbe8c21327c6c1d06a3f1eecb4.sig  | jq
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.oci.image.config.v1+json",
+    "size": 242,
+    "digest": "sha256:ef254edd0dd0ae2b9b6ef0471c915682c0ba31858fe45bc817a951cad715fe8d"
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.dev.cosign.simplesigning.v1+json",
+      "size": 263,
+      "digest": "sha256:f8aae8441b2d41419ae99d4979d6eab1543d81a269e4721fdbff0d837571aa4d",
+      "annotations": {
+        "dev.cosignproject.cosign/signature": "MEUCIGvgIIu9JX5vVCZ5YHsoVe3wpVoIianJvzkCZUIClVlAAiEAyx7GZztNDzCJN5Fjg8vwq0pv7PH3TrseIXN1ou35Jmw=",
+        "dev.sigstore.cosign/bundle": "{\"SignedEntryTimestamp\":\"MEYCIQDsAPhTbHc7w+LHpipbx0HSEIlmjjbAszF5ChHRO1jyKgIhAJrDmeLtzO72fGD1WHabY8Mwz7Qph9CzDg+EMkC9JR7A\",\"Payload\":{\"body\":REDACTED}}",
+        "dev.sigstore.cosign/certificate": "-----BEGIN CERTIFICATE-----\nREDACTED\n-----END CERTIFICATE-----\n",
+        "dev.sigstore.cosign/chain": "-----BEGIN CERTIFICATE-----\nREDACTED=\n-----END CERTIFICATE-----\n-----BEGIN CERTIFICATE-----\nREDACTED\n-----END CERTIFICATE-----"
+      }
+    }
+  ]
+}
+```
+You can see that it contains a layer of type `application/vnd.dev.cosign.simplesigning.v1+json`. Also, its annotation metadata helps cosign to authenticate the signature against the Sigstore's [Rekor transparency log](https://docs.sigstore.dev/logging/overview/).
+
+#### Index update
+
+Each artifact type has its own [registry metadata file](https://github.com/falcosecurity/plugins/blob/master/registry.yaml). The general [falcoctl index.yaml](https://github.com/falcosecurity/falcoctl/blob/gh-pages/index.yaml), that refers to all artifact types, [is updated](https://github.com/falcosecurity/test-infra/blob/bc4db5255ad6f85ee437368bc37ee752f980182d/config/jobs/build-plugins/build-plugins.yaml#L2) when a modification to the specific artifact type metadata occurs. In order to verify a keyless signature we need to know in advance some additional metadata about the type of signature and how to verify its authenticity. We are adding this data to the registry metadata so that it can be published to the index by the pipeline. We will see this content in the next section.
```

Comment:
```suggestion
Each artifact type has its own [registry metadata file](https://github.com/falcosecurity/plugins/blob/master/registry.yaml). The general [falcoctl index.yaml](https://github.com/falcosecurity/falcoctl/blob/gh-pages/index.yaml), which refers to all artifact types, [is updated](https://github.com/falcosecurity/test-infra/blob/bc4db5255ad6f85ee437368bc37ee752f980182d/config/jobs/build-plugins/build-plugins.yaml#L2) when a modification to the specific artifact type metadata occurs. In order to verify a keyless signature we need to know in advance some additional metadata about the type of signature and how to verify its authenticity. We are adding this data to the registry metadata so that it can be published to the index by the pipeline. We will see this content in the next section.
```